File: Native/Test/Python/mr-value-map.py
#
#  Copyright 2010-2012 Fabric Technologies Inc. All rights reserved.
#

import fabric

client = fabric.createClient()

cv = client.MR.createConstValue('String', 'Hello')

vmo = client.KLC.createValueMapOperator("vm.kl", "operator vm( String input, io String output ) { output = input + ', world!'; }", "vm")

vm = client.MR.createValueMap(cv, vmo)

print(vm.produce())

client.close()

File: Native/Test/Node/mr-value-map.js
/*
 *  Copyright 2010-2012 Fabric Engine Inc. All rights reserved.
 */

FC = require("Fabric").createClient();
cv = FC.MR.createConstValue('String', 'Hello');
vmo = FC.KLC.createValueMapOperator("vm.kl", "operator vm( String input, io String output ) { output = input + ', world!'; }", "vm");
vm = FC.MR.createValueMap(cv, vmo);
console.log(vm.produce());
FC.close();
